demo/d3d12/main.c

```diff
@@ -1,4 +1,17 @@
 /* nuklear - 1.32.0 - public domain */
+
+/*
+ *   === IMPORTATE NOTE FOR D3D12 ===  
+ *
+ * Due to a bug in the Windows SDK that
```

Comment:
Please define/describe shortly the bug in the comment so that it is `grep`pable. Then I will merge. Thanks a lot!

Reply:
Oh and maybe squash the commits - it is a small change we could keep together :wink:.